[JavaScript] Create `reference/keywords/const.md` (#285)
exercism/v3: languages/javascript/reference/info/immutability.md

# Immutability

> This is the concept in which a value cannot be changed after having been created.

Reviewer: "This is the concept in " sounds a bit unnatural to me.
How about:

```suggestion
An immutable value is a value that cannot be changed after it has been created.
```
